src/e2e/resources/go/user/user.go.snippet

```diff
@@ -12,28 +12,28 @@ type env struct {
 
 // createTempleuserRequest contains the client-provided information required to create a single templeuser
 type createTempleuserRequest struct {
-	Username           *string    `valid:"-"`
-	Email              *string    `valid:"-"`
-	FirstName          *string    `valid:"-"`
-	LastName           *string    `valid:"-"`
-	CreatedAt          *time.Time `valid:"-"`
-	NumberOfDogs       *int32     `valid:"-"`
-	CurrentBankBalance *float32   `valid:"-"`
-	BirthDate          *time.Time `valid:"-"`
-	BreakfastTime      *time.Time `valid:"-"`
+	Username           *string    `valid:"type(string),required"`
+	Email              *string    `valid:"type(string),required,stringlength(5|40)"`
+	FirstName          *string    `valid:"type(string),required"`
+	LastName           *string    `valid:"type(string),required"`
+	CreatedAt          *time.Time `valid:"type(string),rfc3339WithoutZone,required"`
+	NumberOfDogs       *int32     `valid:"type(int32),required"`
+	CurrentBankBalance *float32   `valid:"type(float32),required"`
+	BirthDate          *time.Time `valid:"type(string),rfc3339WithoutZone,required"`
```

Comment:
Can `rfc3339WithoutZone` distinguish between date and datetime, and do we want to make that distinction?